### PR TITLE
Adds Spectra and KSPRC (terrain) textures.

### PR DESCRIPTION
**Pull request by MoreRobustThanYou**
capture

### CODE REVIEW

File: NetKAN/KSPRC-Textures.netkan [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	"spec_version": "v1.2",
+	"name": "KSPRC Textures",
+	"abstract": "Standalone KSPRC Textures. Kinda useless without proper configs. (CC-BY-ND-4.0)",
+	"identifier": "KSPRC-Textures",
+	"download": "https://spacedock.info/content/Proot_4811/KSPRC_-_Renaissance_Compilation_artworks_remake/KSPRC_-_Renaissance_Compilation_artworks_remake-0.7_PreRelease_3.zip",
+	"version": "0.7",
+	"release_status": "stable",
+	"ksp_version_min": "1.3",
+	"ksp_version_max": "1.3.99",
+	"x_netkan_license_ok": true,
```
Comment (Olympic1) [inline NetKAN/KSPRC-Textures.netkan]: If you're using `x_netkan_license_ok`, you need a `license`

Comment (Olympic1) [reply NetKAN/KSPRC-Textures.netkan]: It needs a `license`
Add `"license": "CC-BY-ND-4.0",`

Comment (MoreRobustThanYou) [reply NetKAN/KSPRC-Textures.netkan]: CC-BY-ND-4.0 isn't in the schema, see #2160

Comment (MoreRobustThanYou) [reply NetKAN/KSPRC-Textures.netkan]: but ok i added dat

**Comment by MoreRobustThanYou** [comment]
btw if this is merged, ya might wanna squash it

**Comment by MoreRobustThanYou** [comment]
@Olympic1 why is the KSPRC-Textures failing wtf

**Comment by Olympic1** [comment]
> btw if this is merged, ya might wanna squash it

We always squash the commits.

> why is the KSPRC-Textures failing wtf

Add `"author": "Proot",`

**Comment by MoreRobustThanYou** [comment]
@Olympic1 ok done, can it be merged if it compiles successfully?

**Comment by MoreRobustThanYou** [comment]
reeee

```CKAN.InvalidKSPInstanceKraken```

File: NetKAN/KSPRC-Textures.netkan (hunk 1 above)
Comment (MoreRobustThanYou) [reply NetKAN/KSPRC-Textures.netkan]: still not working @Olympic1 even after adding author and license

**Comment by MoreRobustThanYou** [comment]
Reee it nearly works now, except Spectra uses a TRR .dds file that TRR also installs

**Comment by politas** [comment]
You'll need to add a `"filter": [ "files", "that", "should", "be", "ignored" ]` into the install instructions so that you aren't over-writing the dependency's files.

**Comment by MoreRobustThanYou** [comment]
YESSS IT WORKS NOW

**Comment by MoreRobustThanYou** [comment]
@Olympic1 can you merge?

**Comment by MoreRobustThanYou** [comment]
thx

**Pull request merged**